apps/emqx/test/emqx_cth_cluster.erl

```diff
@@ -17,9 +17,11 @@
 -module(emqx_cth_cluster).
```

Comment:
I think it is better to add a short summary for each module. 
I can't guess what the meaning of `cth` out, sorry.

Reply:
@keynslug Will add a description to this module. Since it had been added before this PR was opened I think it's a different PR.